docs/commands/endpoint.adoc

```diff
@@ -44,14 +44,6 @@ Parameters are the following elements:
 
 The technical message type can be a technical message type of DKE or a proprietary message type.
 
-[IMPORTANT]
```

Comment:
Why did you remove this part? Did this move to another location? If yes, feel free to resolve. As far as I know this behavior still persists, or does it not?